### PR TITLE
Reset button in UE

### PR DESCRIPTION
**Pull request by TalmizAhmed**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--aem-boilerplate-forms--adobe-rnd.hlx.live/
- After: https://ue-reset--aem-boilerplate-forms--adobe-rnd.hlx.live/

### CODE REVIEW

**Review by nit23uec — DISMISSED** [review]

File: component-definition.json [hunk 1]
```diff
@@ -273,6 +273,22 @@
               }
             }
           }
+        },
+        {
+          "title": "Reset",
```
Comment (jalagari) [inline component-definition.json:278]: do we have test cases for reset?

File: component-definition.json [hunk 2]
```diff
@@ -273,6 +273,22 @@
...
+          "id": "form-reset-button",
```
Comment (nit23uec) [inline component-definition.json:279]: this would also need to be there in filter

File: component-definition.json (hunk 1 above)
Comment (nit23uec) [reply component-definition.json:278]: this PR inn itself does not includes any code for including the component in UE (it's taken care by the UE layer itself).
the test case of reset button should be covered by form-block runtime code. (its there in af2-web-runtime)

File: component-definition.json (hunk 2 above)
Comment (TalmizAhmed) [reply component-definition.json:279]: Added both submit and reset

**Review by vdua — APPROVED** [review]

**Pull request merged**